db/data/20231018062055_re_enable_sms_india.rb

```diff
@@ -0,0 +1,74 @@
+# frozen_string_literal: true
+
+class ReEnableSmsIndia < ActiveRecord::Migration[6.1]
+  PATIENTS_PER_DAY = 5000
+  FILTERS = {
+    "states" => {"include" => ["Maharashtra", "Andhra Pradesh", "West Bengal"]}
+  }.freeze
+  CANCELLED_EXPERIMENTS_MONTHS = (9..12).map do |month_number|
+    Date::ABBR_MONTHNAMES[month_number]
+  end
+
+  NEW_EXPERIMENTS_DATA = (11..12).map do |month_number|
+    month = Date::ABBR_MONTHNAMES[month_number]
+    {
+      start_time: DateTime.new(2023, month_number).beginning_of_month,
+      end_time: DateTime.new(2023, month_number).end_of_month,
+      current_patients_experiment_name: "Current Patient V2 #{month} 2023",
+      stale_patients_experiment_name: "Stale Patient V2 #{month} 2023"
+    }
+  end
+
+  def up
+    return unless CountryConfig.current_country?("India") && SimpleServer.env.production?
+
+    CANCELLED_EXPERIMENTS_MONTHS.each do |month|
+      outdated_current_experiment = Experimentation::CurrentPatientExperiment.find_by_name("Current Patient #{month} 2023")
+      outdated_current_experiment.cancel
+      outdated_current_experiment.evict_patients
```

Comment:
This isn't necessary since `evict_patients` only conditionally evicts treatment group memberships. The patients in these experiments will also be available to the next experiment considering the amount of time between them.